tests-integration/tests/lockdrop.rs

```diff
@@ -69,7 +69,7 @@ fn test_lockdrop() {
         Milliseconds::from_seconds(current_timestamp),
         Milliseconds::from_seconds(100u64),
         Milliseconds::from_seconds(50u64),
-        cw20_incentives_address.to_string(),
+        AndrAddr::from_string(cw20_incentives_address.to_string()),
```

Comment:
Can we try using a VFS path instead of the actual address?

Reply:
Updated to use VFS path. Found a bug by updating the use case. 👍